lib/web_ui/lib/src/engine/platform_dispatcher.dart

```diff
@@ -92,6 +93,10 @@ class EnginePlatformDispatcher extends ui.PlatformDispatcher {
   /// The [EnginePlatformDispatcher] singleton.
   static EnginePlatformDispatcher get instance => _instance;
   static final EnginePlatformDispatcher _instance = EnginePlatformDispatcher();
+  static final DomElement _accessibilityPlaceholder = EngineSemantics
```

Comment:
Can we remove `static`? The `PlatformDispatcher` is already a singleton that should guarantee there's no more than one placeholder. However, being non-static would guarantee that the lifecycle of the placeholder is attached to the lifecycle to `PlatformDispatcher`, such as when hot-reload/hot-restart happens, or when/if we support shutting down and reinitializing the engine.